.github/workflows/build_charm_without_cache.yaml

```diff
@@ -0,0 +1,59 @@
+# Copyright 2023 Canonical Ltd.
+# See LICENSE file for licensing details.
+
+# Usage documentation: build_charm_without_cache.md
+
+on:
+  workflow_call:
+    inputs:
+      artifact-name:
+        description: Packed charm is uploaded to this GitHub artifact name
+        default: charm-packed-without-cache
+        type: string
+      path-to-charm-directory:
+        description: Relative path to charm directory from repository directory
+        default: .
+        type: string
+    outputs:
+      artifact-name:
+        description: Packed charm is uploaded to this GitHub artifact name
+        value: ${{ inputs.artifact-name }}
+
+jobs:
+  build:
+    name: Build charm
+    runs-on: ubuntu-latest
+    timeout-minutes: 60
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+      - name: Setup environment
+        # TODO: Replace with custom image on self-hosted runner
+        run: |
+          # Copied from https://github.com/charmed-kubernetes/actions-operator/blob/96fb0b07eb675f74cf1796be812bc7e67a0d62fc/src/bootstrap/index.ts#L151
+          sudo adduser $USER lxd
+          newgrp lxd
+          sudo lxd waitready
+          sudo lxd init --auto
+          sudo iptables -F FORWARD
+          sudo iptables -P FORWARD ACCEPT
+          sudo chmod a+wr /var/snap/lxd/common/lxd/unix.socket
```

Comment:
excepted line #37, is this a super safe way to enable lxd or just unnecessary/obsolete? I don't remember seeing such routine just to have LXD running